### PR TITLE
Uml 1533 hashing logged email

### PR DESCRIPTION
**Pull request by Lbagg1**
# Purpose
Currently we hash email addresses that we are logging to protect PII from being leaked. 

There are 2 places where we have missed this:

UserService.php (front) line 135 

$this->logger->notice(
    'Authentication failed for {email} with code
        [
        'code'  => $e->getCode(),
        'email' => $email
    ]
);
 

UserService.php (api) line 200

$this->logger->notice(
    'Attempt made to reset password for non-existent account',
    ['email' => $email]
);
 

AC

The email is hashed in these places where we have missed it

Search code base for other missed examples and fix

Including MLPA - LPAL-1264: Hash PII when logging password reset requests
BACKLOG
 

Fixes UML-1533

## Checklist

* [ ] I have performed a self-review of my own code
* ~[ ] I have added relevant logging with appropriate levels to my code~
* ~[ ] New event_codes have been documented on the [wiki page](https://opgtransform.atlassian.net/wiki/spaces/LSML2/pages/3277881441/Understanding+the+event+logs)~
*~ [ ] I have updated documentation (Confluence/GitHub wiki/tech debt doc) where relevant~
* [ ] I have added tests to prove my work
* ~[ ] I have added welsh translation tags and updated translation files~
*~ [ ] I have run an accessibility tool on any pages I have made changes to and fixed any issues found~
* ~[ ] I have notified the Interaction Designer of any content changes so that appropriate screenshots/flow diagram changes can be made~
* [ ] The product team have tested these changes

### CODE REVIEW

File: service-api/.idea/.gitignore
Comment (cooperaj) [inline service-api/.idea/.gitignore:1]: Please remove the commited .idea folder here. There should only be one of these at the top level of the project.

File: service-api/app/src/App/src/Service/User/UserService.php [hunk 1]
```diff
@@ -190,7 +190,7 @@ public function requestPasswordReset(string $email): array
         } catch (Exception $e) {
             $this->logger->notice(
                 'Attempt made to reset password for non-existent account',
-                ['email' => $email]
+                ['email' => new Email($email)]
```
Comment (mattmachell) [inline service-api/app/src/App/src/Service/User/UserService.php:193]: Not a full namespace and not appearing added as an include?

**Review by cooperaj — CHANGES_REQUESTED** [review]
Looks good bar the .idea folder 👍

File: service-api/app/src/App/src/Service/User/UserService.php (hunk 1 above)
Comment (cooperaj) [reply service-api/app/src/App/src/Service/User/UserService.php:193]: Good catch.

**Review by cooperaj — DISMISSED** [review]

**Review by cooperaj — APPROVED** [review]

**Pull request merged**